build/compiler.py

```diff
@@ -353,10 +353,16 @@ def lint(self, fix=False, force=False):
 
     Returns:
       True on success; False on failure.
-    """
+    """  
     deps = self.source_files + [self.config_path]
     if not force and not _must_build(self.output, deps):
       return True
+    
+    """Windows shows an error when the file location has '\\' so for windows the below if condition changes the'\\' with '/' in variable storing file locations """
+    if os.name == 'nt' :
```

Comment:
A better check would be `sys.platform == 'win32'`.

Reply:
I did change the if condition as recommended . Out of curiosity , is there a specific reason for choosing sys.platform over os.name other than value 'win32' , 'nt' where 'win32' more clearly indicates windows .

Reply:
Looking at the docs again, I was mistaken on which is more standardized.  It seems that `sys.platform` is a bit more fine-grained as it basically uses `uname` command.  But `os.name` returns only spec-defined values.  We use `sys.platform` more often, so it is probably better to just be consistent.